lib/CMakeLists.txt

```diff
@@ -151,8 +151,14 @@ endfunction()
 # STATIC
 #   Build a static library.
 #
-# LLVM_LINK_COMPONENTS
-#   LLVM components this library depends on.
+# EMIT_MODULE
+#   Emit '.swiftmodule' to
+#
+# DEPENDENCIES
+#   Target names to pass target_link_library
+#
+# FORCE_DEPENDENCIES
```

Comment:
You could maybe name this SWIFT_DEPENDENCIES instead. Ideally with a comment saying it'll be removed when we fix CMake.

Reply:
:+1: Renamed to SWIFT_DEPENDENCIES